src/codeu/chat/util/Tokenizer.java

```diff
@@ -0,0 +1,100 @@
+/*
+* This class parses a string to retrive tokens
+* and determines whether they are surrounded by qoutations or not
+*
+* author: Isaac Martins
+* version: May 19, 2017
+*/
+
+public final class Tokenizer
```

Comment:
(The same comments about spaces after keywords and about opening curly-brace placement apply here too.)